docs/README.md

```diff
@@ -0,0 +1,155 @@
+# KFServing
+A KFService is a "unit of model serving". Users may have many KFServices, including different definitions across dev, test, and prod environments, as well as across different model use cases. KFServices are defined using the [Kubernetes Custom Resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) standard. 
```

Comment:
How about highlight kfserving is based on kative serving? This can help the dev/user understand this project easily.

Reply:
We're intentionally decoupling Knative from our documentation. It's important that knative is just an implementation detail to maintain flexibility if we end up not being able to use it or want to support other options.

Reply:
@ellis-bigelow thanks - the issue with decoupling too much is that it becomes an entire new nomenclature to understand, use and work towards. The goal should be to have a very little disruption for knative users, if that`s the primary backing tech here, so that artifacts, ecosystem which emerges around it can be readily reused. 

I gave a talk last Kuebcon around the potential of using knative for ML pipelines, and got great feedback from core kube users because it was explicitly stating the knative can be used, and in what ways. 
https://www.slideshare.net/AnimeshSingh/ai-machine-learning-pipelines-with-knative

Reply:
I saw your talk!

Feel free to come to one of our KFServing Weekly syncs to bring this up. Just let me know when you'd like to discuss and I'll add it to our agenda. This issue is reasonably contentious, and this is probably the wrong forum to debate it :)

Reply:
Thanks @ellis-bigelow. 

Do we have one next week - can sync up there. I am seeing this message - 'This event has been canceled and removed from your calendar.'

Reply:
Every Wednesday. Fridays are special topics and this weeks is cancelled.